extensions/amp-a4a/amp-a4a-format.md

```diff
@@ -0,0 +1,392 @@
+  <tr><td>amp-list</td></tr>
+  <tr><td>amp-live-list</td></tr>
+  <tr><td>amp-o2-player</td></tr>
+  <tr><td>amp-pinterest</td></tr>
+  <tr><td>amp-reach-player</td></tr>
+  <tr><td>amp-share-tracking</td></tr>
+  <tr><td>amp-slides</td></tr>
+  <tr><td>amp-social-share</td></tr>
+  <tr><td>amp-soundcloud</td></tr>
+  <tr><td>amp-springboard-player</td></tr>
+  <tr><td>amp-sticky-ad</td></tr>
+  <tr><td>amp-twitter</td></tr>
+  <tr><td>amp-vimeo</td></tr>
+  <tr><td>amp-vine</td></tr>
+  <tr><td>amp-youtube</td></tr>
+</table>
+
+### HTML Tags
+
+The following are _allowed_ tags in an A4A creative.  Tags not explicitly 
+allowed are prohibited.  This list is a subset of the general [AMP tag 
+addendum whitelist](../../spec/amp-tag-addendum.md). Like that list, it is 
+ordered consistent with HTML5 spec in section 4 [The Elements of HTML](http://www.w3.org/TR/html5/single-page.html#html-elements).
+
+Most of the omissions are either for performance or because the tags are not
+HTML5 standard.  For example, `<noscript>` is omitted because A4A depends on
+JavaScript being enabled, so a `<noscript>` block will never execute and,
+therefore, will only bloat the creative and cost bandwidth and latency. 
+Similarly, `<acronym>`, `<big>`, et al. are prohibited because they are not
+HTML5 compatible.
+
+#### 4.1 The root element
+4.1.1 `<html>`
+  - Must use types `<html a4⚡>` or `<html a4amp>`
+
+#### 4.2 Document metadata
+4.2.1 `<head>`  
+4.2.2 `<title>`  
+4.2.4 `<link>`  
+  - `<link rel=...>` tags are disallowed, except for `<link rel=stylesheet>`.
+
+4.2.5 `<meta>`
+  - Only `<meta charset=utf8>` and `<meta name=viewport>` are allowed.
+
+4.2.6 `<style>`  
+
+#### 4.3 Sections
+4.3.1 `<body>`  
+4.3.2 `<article>`  
+4.3.3 `<section>`  
+4.3.4 `<nav>`  
+4.3.5 `<aside>`  
+4.3.6 `<h1>`, `<h2>`, `<h3>`, `<h4>`, `<h5>`, and `<h6>`  
+4.3.7 `<header>`  
+4.3.8 `<footer>`  
+4.3.9 `<address>`  
+#### 4.4 Grouping Content
+4.4.1 `<p>`  
+4.4.2 `<hr>`  
+4.4.3 `<pre>`  
+4.4.4 `<blockquote>`  
+4.4.5 `<ol>`  
+4.4.6 `<ul>`  
+4.4.7 `<li>`  
+4.4.8 `<dl>`  
+4.4.9 `<dt>`  
+4.4.10 `<dd>`  
+4.4.11 `<figure>`  
+4.4.12 `<figcaption>`  
+4.4.13 `<div>`  
+4.4.14 `<main>`  
+#### 4.5 Text-level semantics
+4.5.1 `<a>`  
+4.5.2 `<em>`  
+4.5.3 `<strong>`  
+4.5.4 `<small>`  
+4.5.5 `<s>`  
+4.5.6 `<cite>`  
+4.5.7 `<q>`  
+4.5.8 `<dfn>`  
+4.5.9 `<abbr>`  
+4.5.10 `<data>`  
+4.5.11 `<time>`  
+4.5.12 `<code>`  
+4.5.13 `<var>`  
+4.5.14 `<samp>`  
+4.5.15 `<kbd >`  
+4.5.16 `<sub>` and `<sup>`  
+4.5.17 `<i>`  
+4.5.18 `<b>`  
+4.5.19 `<u>`  
+4.5.20 `<mark>`  
+4.5.21 `<ruby>`  
+4.5.22 `<rb>`  
+4.5.23 `<rt>`  
+4.5.24 `<rtc>`  
+4.5.25 `<rp>`  
+4.5.26 `<bdi>`  
+4.5.27 `<bdo>`  
+4.5.28 `<span>`  
+4.5.29 `<br>`  
+4.5.30 `<wbr>`  
+#### 4.6 Edits
+4.6.1 `<ins>`  
+4.6.2 `<del>`  
+#### 4.7 Embedded Content
+- Embedded content is supported only via AMP tags, such as `<amp-img>` or 
+`<amp-video>`.
+
+#### 4.7.8
+4.7.8 `<source>`  
+
+#### 4.7.15 SVG
```

Comment:
-Inline SVG
We currently have 20kb max file size for CSS.  Instead, maybe we should have a max file size/initial load that includes images, because multiple inline SVGs in the HTML can make the file size too big.